recipes/tafra/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "tafra" %}
+{% set version = "1.0.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a0affd16c6c47123cb2832fcd8dc11d0e6b8a02752d3368f63a8060b2fd431f0
+
+build:
+  number: 3
```

The build number should only be incremented when a binary has already been uploaded to conda-forge:
```suggestion
  number: 0
```